scripts/.eslintrc

```diff
@@ -2,15 +2,10 @@
   "env": {
     "node": true
   },
-  "extends": [
-    "../.eslintrc",
```

Comment:
@suevalov I might be missing something, but didn't we lose all the default eslint and promise plugins checks here?